src/main/java/apoc/export/util/ExportConfig.java

```diff
@@ -71,10 +76,12 @@ public ExportConfig(Map<String,Object> config) {
         config = config != null ? config : Collections.emptyMap();
         this.silent = toBoolean(config.getOrDefault("silent",false));
         this.batchSize = ((Number)config.getOrDefault("batchSize", DEFAULT_BATCH_SIZE)).intValue();
-        this.delim = delim(config.getOrDefault("d", String.valueOf(DEFAULT_DELIM)).toString());
+        this.delim = delim(config.getOrDefault("delim", String.valueOf(DEFAULT_DELIM)).toString());
```

Comment:
perché abbiamo cambiato questa property?

Reply:
Perchè in documentazione era scritta come `delim` e nella classe di export era scritta come `d`. nel caso in cui si fosse messo la opzione di `delim` non l'avrebbe presa